launch_testing/package.xml

```diff
@@ -3,26 +3,21 @@
 <package format="2">
   <name>launch_testing</name>
   <version>0.7.3</version>
-  <description>Helper scripts for tests that use the ROS launch tool.</description>
+  <description>ROS launch testing suite.</description>
```

Comment:
Should match the `setup.py` and not mention ROS.

Reply:
Matched `long_description` attribute in the `setup.py`.